play.py

```diff
@@ -186,3 +213,6 @@ def choose_first_player(model1, model2):
     else:
         other_model, current_model = model1, model2
     return current_model, other_model
+def decide_disable_resign():
+    return random() < .1
```

Comment:
The .1 should be in a configuration variable. 

if the variable has a good name, I feel the need for a function might not be needed as much

`if decide_disable_resign():
`
vs

`if random() < RESIGNATION_DISABLE_PERCENT:`